modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/commands/openCategorySelectionModal.ts

```diff
@@ -46,23 +48,29 @@ export default function openCategorySelectionModal({
 			if (!Object.keys(selectedItems).length) {
 				return;
 			}
+			if (onSelectMethod) {
+				onSelectMethod(selectedItems);
+			}
+			else {
```

Comment:
I don't want to overcomplicate the module, our current usecase is satisfied with the change I made, and in case we need to have both the onselect and navigateOnSelection to run, we can make that change